config/sidekiq.yml

```diff
@@ -10,18 +10,19 @@
 
 :schedule:
   CalculateAllMetrics:
-    cron: '0 0 * * * *' # Run at midnight
+    cron: '0 0 0 * * *' # Run at midnight
```

Comment:
Here comment are not representative of the current cron config 
'0 0 0 * * *' => "At 12:00 AM, on day 1 of the month"
https://crontab.cronhub.io/